maquetado/scss/fonts.scss

```diff
@@ -0,0 +1,5 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
```

Comment:
This is not a font constant, more a specific one. I would put this in the same file you styled the title element